File: public/usage-examples/graphics/draw_triangle_on_window/Simple_Red_Triangle.cpp
#include "splashkit.h"

int main()
{
    open_window("Simple Red Triangle", 400, 300);

    while (not quit_requested())
    {
        process_events();
        clear_screen(COLOR_WHITE);
        fill_triangle(COLOR_RED, 100, 100, 200, 100, 150, 200);
        refresh_screen(60);
    }

    return 0;
}

File: public/usage-examples/graphics/draw_triangle/Simple_Red_Triangle.py
from splashkit import *

def main():
    window = create_window("Simple Red Triangle", 800, 600)

    while not window.close_requested:
        process_events()
        clear_screen(color_white())
        fill_triangle(color_red(), 400, 100, 600, 500, 200, 500)
        refresh_window(window)

    close_window(window)

if __name__ == "__main__":
    main()
